### DISCUSSION TITLE
unicode parameter names support does not work(

### DISCUSSION THREAD

**Discussion by lislisl** [scikit-hep/iminuit#697] [Q&A]
Dear Developers,
I am trying to run basic_tutorial notebook https://github.com/scikit-hep/iminuit/blob/develop/doc/tutorial/basic_tutorial.ipynb
on swan.cern.ch,
iminuit is installed,
and I've got Unicode Encode Error (see below) on the third cell.
Are there some additional libraries that need to be installed?
With best regards,
Sofia

---------------------------------------------------------------------------
UnicodeEncodeError                        Traceback (most recent call last)
/tmp/ipykernel_880/3214520957.py in <module>
      3 least_squares = LeastSquares(data_x, data_y, data_yerr, line)
      4 
----> 5 m = Minuit(least_squares, α=0, β=0)  # starting values for α and β
      6 
      7 m.migrad()  # finds minimum of least_squares function

src/iminuit/_libiminuit.pyx in iminuit._libiminuit.Minuit.__init__()

src/iminuit/_libiminuit.pyx in iminuit._libiminuit.Minuit._init_pyfcn()

/cvmfs/sft.cern.ch/lcg/views/LCG_101swan/x86_64-centos7-gcc8-opt/lib/python3.9/site-packages/iminuit/_libiminuit.cpython-39-x86_64-linux-gnu.so in vector.from_py.__pyx_convert_vector_from_py_std_3a__3a_string()

/cvmfs/sft.cern.ch/lcg/views/LCG_101swan/x86_64-centos7-gcc8-opt/lib/python3.9/site-packages/iminuit/_libiminuit.cpython-39-x86_64-linux-gnu.so in string.from_py.__pyx_convert_string_from_py_std__in_string()

UnicodeEncodeError: 'ascii' codec can't encode character '\u03b1' in position 0: ordinal not in range(128)

**Comment by HDembinski** [comment]
Which iminuit version are you using? It looks like you have the iminuit 1.x series installed, not the current 2.x series. iminuit 1.x did not support unicode characters.

**Reply by HDembinski** [reply]
I am moving this to the issues.
